src/Hooks/DynamicElements.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace WPML\PB\Elementor\Hooks;
+
+class DynamicElements implements \IWPML_Frontend_Action, \IWPML_DIC_Action {
```

Comment:
@strategio I don't think you need `IWPML_DIC_Action` here because the class has no dependencies that I can see.

Reply:
@brucepearson, do you see a problem using the DIC instead of direct instantiation? Because I tend to always use the DIC even if the class does not have any dependeny.

Now the class instantiation with the DIC is covered with an integration test and we don't need to care anymore about it if we decide to extend the class later (and introduce a dependency).

Reply:
DIC will be a little slower but I don't mind either way. I just mentioned it just in case you added it by mistake or thought it was needed.